### DISCUSSION TITLE
Worked perfectly

### DISCUSSION THREAD

**Discussion by gjohnz** [rhinstaller/anaconda-webui#42] [General]
The webui worked flawlessly when use to install a virtual machine.

Host was Fedora 37 Workstation. VM is VirtualBox 7.0.0

Installed the OS, rebooted and ran a full update. No issues what so ever.

**Comment by jkonecny12** [comment]
Great you liked it. Thanks for saying!

**Reply by rexhent** [reply]
I also had no issues on qemu/kvm.
